crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -71,6 +71,10 @@ pub struct AuthorityPerpetualTables {
     pub(crate) effects: DBMap<TransactionEffectsDigest, TransactionEffects>,
     pub(crate) synced_transactions: DBMap<TransactionDigest, TrustedCertificate>,
 
+    /// When transaction is executed via checkpoint executor, we store association here
+    pub(crate) executed_transactions_to_checkpoint:
+        DBMap<TransactionDigest, (EpochId, CheckpointSequenceNumber)>,
```

Comment:
what might our pruning strategy be for this table?

Reply:
I don't have any pruning strategy in mind unfortunately.
You can create transaction in epoch 1 million pointing to txn in epoch 0, and we need to know about it...